packages/backend-auth/src/factory.ts

```diff
@@ -195,6 +203,7 @@ class AmplifyAuthGenerator implements ConstructContainerEntryGenerator {
           policy.attachToRole(role);
         },
       }),
+      stack: Stack.of(this.generateContainerEntry.prototype),
```

Comment:
having trouble with this, if I use t`his.generateContainerEntry.prototype.scope`, tests fail because `scope` could be undefined. If I use `Stack.of(this.generateContainerEntry.prototype)`, tests fail because `Stack.of()` is undefined/cannot be read. I can't get rid of this or make it undefined because this is how the `stack` parameter is able to be called in `backend.auth.stack`